www/js/viewer.js

```diff
@@ -280,10 +283,14 @@ const createCards = (rows, LineID) => {
 
     gurmukhiContainer.innerHTML = `<span class="padchhed">${taggedGurmukhi.join(' ')}</span>
                                     <span class="larivaar">${taggedGurmukhi.join('<wbr>')}</span>`;
+
+    const englishContainer = document.createElement('div');
+    englishContainer.innerHTML = row.English;
```

Comment:
should make this `row.English || ''` to avoid 
![image](https://user-images.githubusercontent.com/2982784/57713445-5a29c600-7627-11e9-977b-21cae2283ea3.png)